eslint.config.mjs

```diff
@@ -321,6 +327,26 @@ export default typescript.config([
       'import/no-unresolved': 'off', // Disabled in favor of typescript-eslint
     },
   },
+  {
+    name: 'plugin/filename-export',
+    // https://github.com/ekwoka/eslint-plugin-filename-export?tab=readme-ov-file#rules
+    plugins: {'filename-export': filenameExport},
+    rules: {
+      'filename-export/match-named-export': 'off', // This is a hard one to enable, we have loads of files like: utils.tsx, styles.tsx, data.tsx...
+      'filename-export/match-default-export': ['off', {casing: 'loose'}], // TODO(ryan953): Fix violations and enable this rule
+    },
+  },
```

Comment:
this feels aspirational

Reply:
A little.

There are a few where the names are sooooo close to matching. This will be nice to surface that.

Also I thought naming was hard... why do people pick two different names for things?

Reply:
sometimes the folder is already named Project and you're making a list view so calling the file Project/ProjectList.tsx is repetitive

Reply:
but we've got real cases like this which are dumb, we could be picking better names overall. and yes i made these files:
- static/app/views/replays/list/replaysList.tsx
- static/app/views/replays/list.tsx